salesapp.js

```diff
@@ -46,15 +68,19 @@ CookieStore.prototype.populateTable = function() {
   rowEl.appendChild(fieldTotal);
   tableEl.appendChild(rowEl);
 };
+//-----------------------------------VARIABLES----------------------------
+var pikePlace = new CookieStore('Pike Place', 23, 65, 6.3);
+var seaTac = new CookieStore('SeaTac', 3, 24, 1.2);
+var seaCenter = new CookieStore('Seattle Center', 11, 38, 3.7);
+var capHill = new CookieStore('Capitol Hill', 20, 38, 2.8);
+var alKai = new CookieStore('Alkai', 2, 16, 4.6);
+
+var storeArray = [pikePlace, seaTac, seaCenter, capHill, alKai, formStore];
 
-var pikePlace = new CookieStore('pike', 'Pike Place', 23, 65, 6.3);
-var seaTac = new CookieStore('seatac', 'SeaTac', 3, 24, 1.2);
-var seaCenter = new CookieStore('seacent', 'Seattle Center', 11, 38, 3.7);
-var capHill = new CookieStore('caphill', 'Capitol Hill', 20, 38, 2.8);
-var alKai = new CookieStore('alkai', 'Alkai', 2, 16, 4.6);
-var storeArray = [pikePlace, seaTac, seaCenter, capHill, alKai];
 var storeTimes = ['Location', '6:00am', '7:00am', '8:00am', '9:00am', '10:00am', '11:00am', '12:00pm', '1:00pm', '2:00pm', '3:00pm', '4:00pm', '5:00pm', '6:00pm', '7:00pm', '8:00pm', 'Total'];
```

Comment:
Global, hard-coded data like this is probably best put up at the very top of the file, because everything else sort of depends on it.